src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -72,6 +72,18 @@ export function ConfigEditor(props: Props) {
     }
   }, [arn]);
 
+  // Description to show in datasource card in list
+  const { dbUser, clusterIdentifier, database } = props.options.jsonData || {};
+  useEffect(() => {
+    console.log(`${dbUser}@${clusterIdentifier}/${database}`);
+    if (dbUser || clusterIdentifier || database) {
+      props.onOptionsChange({
+        ...props.options,
+        url: `${dbUser}@${clusterIdentifier}/${database}`,
```

Comment:
As mentioned in PR desc, two questions here:

1. Is it okay to reuse the `url` field? A better way could be to create a details optional prop in the Grafana `DataSourceSettings` interface and use that instead to make it clear we are not restricted to URLs, but that seemed like a big change... thoughts?

2. Also happy to change the formatting (`user@cluster/db`) if you think of something better, one concern could be if one of the three is not populated we will have something like `user@/db`. Maybe we could go for a simpler `user: testUser, cluster: testCluster, database: testDB` ?

Reply:
I think we can make this a real URL, similar to what we do for PostgreSQL. If you go to the cluster settings in the aws console ([link](https://us-east-2.console.aws.amazon.com/redshiftv2/home?region=us-east-2#cluster-details?cluster=redshift-cluster-grafana)), you can see an Endpoint there that looks like: `cluster.stuff....com:5439/db`. We can try and show the same information here. It requires some backend changes though.

The target would be to use this API: https://docs.aws.amazon.com/redshift/latest/APIReference/API_DescribeClusters.html. This is the one that has the `Endpoint` info that we want. To achieve this, we need:

- Add a new Client in the API struct: https://github.com/grafana/redshift-datasource/blob/main/pkg/redshift/api/api.go#L54. At the moment, we only have a `redshiftdataapiservice`  and `secretsmanager` clients, but the target API method (`DescribeClusters`) is in the `redshift` API so we need to add a third client. 
- Add a new method to get the cluster endpoint. It can look very similar to the [Secret](https://github.com/grafana/redshift-datasource/blob/main/pkg/redshift/api/api.go#L321) method but internally using the DescribeClusters method instead.
- Extend the `RedshiftDatasourceIface` with the method we just added and create the function that just calls to the api function (again, just like `Secret`).
- Add a resource route (as with the [secret](https://github.com/grafana/redshift-datasource/blob/main/pkg/redshift/routes/routes.go#L25)). This will be the endpoint our frontend will hit to get the information.

Those are all the backend steps! now for the frontend, we just need to fetch that URL as we do for the secret and save that info here.

If you want to pair for this one, feel free to schedule some time :)